doc/langref.html.in

```diff
@@ -9804,8 +9804,8 @@ pub fn main() !void {
       {#header_close#}
       {#header_open|Builtin Overflow Functions#}
       <p>
-      These builtins return a {#syntax#}bool{#endsyntax#} of whether or not overflow
-      occurred, as well as returning the overflowed bits:
+      These builtins return a {#link|Tuples#} containing wether there was an overflow
+      (as a {#syntax#}u1{#endsyntax#}) and the overflowed bits of the operation:
```

Comment:
```suggestion
      These builtins return a tuple with the result of the respective operation and a possible overflow bit:
```

Reply:
The changes look ok to me.

Reply:
There are a number of things wrong with your changes so I suggested a new simpler sentence altogether.

Reply:
Sorry, I meant that I liked your suggestion.

Should I apply your patch or do something?

Reply:
Ooh, I got you. Yes, if you agree with my suggestion you can commit my suggestion to your branch if you want.

I'm just a contributor like you so I have no permission of committing changes to your git branch.

Here's an alternative suggestion I have:
```suggestion
      These builtins return a tuple containing whether there was an overflow
      (as a {#syntax#}u1{#endsyntax#}) and the possibly overflowed bits of the operation:
```
You can commit either of these if you want to.